runatlantis.io/.vitepress/sidebars.ts

```diff
@@ -98,7 +98,7 @@ const en = [
             {text: "Events Controller", link: "/contributing/events-controller"},
         ]
       },
-      {text: "Glossry", link: "/contributing/glossary"},
+      {text: "Glosssry", link: "/contributing/glossary"},
```

Comment:
should be Glossary?
```suggestion
      {text: "Glossary", link: "/contributing/glossary"},
```

Reply:
Ugh, I do it once, and then have to go fork and fix, and.. make the typo!